api-references/data/insights.json

```diff
@@ -1,1177 +1,2682 @@
+        "parameters": [
+          {
+            "description": "Query details",
+            "name": "params",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/requests.ListETFTransactionsRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/responses.ListETFTransactionsResponse"
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "schema": {
+              "$ref": "#/definitions/responses.Error"
+            }
+          }
+        }
+      }
+    },
+    "/v1/mutual-funds/transactions": {
+      "post": {
+        "description": "List mutual-funds transactions",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "tags": [
+          "List transactions"
+        ],
+        "summary": "List mutual-funds transactions",
+        "operationId": "list-mutual-funds-transactions",
+        "parameters": [
+          {
+            "description": "Query details",
+            "name": "params",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/requests.ListMutualFundsTransactionsRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/responses.ListMutualFundsTransactionsResponse"
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "schema": {
+              "$ref": "#/definitions/responses.Error"
+            }
+          }
+        }
+      }
+    },
+    "/v1/purge": {
+      "delete": {
+        "description": "Purge account and transactions data",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "tags": [
+          "Purge"
+        ],
+        "summary": "Purge data",
+        "operationId": "data-purge",
+        "parameters": [
+          {
+            "description": "purging identifiers",
+            "name": "params",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/requests.PurgeDataRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/responses.PurgeDataResponse"
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "schema": {
+              "$ref": "#/definitions/responses.Error"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
```

Comment:
There is no property `definitions` or `securityDefinitions` in OpenAPI v3.

<img width="396" alt="Screenshot 2024-09-04 at 09 41 49" src="https://github.com/user-attachments/assets/a4997fe4-9ccd-4710-87db-9c46ce7bb975">

Reply:
This has been fixed. Moved the new schemas to `components/schemas`.